node/node.go

```diff
@@ -124,6 +132,15 @@ func (n *Node) RegisterAPI(endpoint string, api http.Handler) error {
 	return nil
 }
 
+// Registers RPC handlers.
+// Other handlers will be added later.
+func (n *Node) RegisterHandlers() error {
+	return n.RegisterAPI("/status", handlers.StatusHandler{
+		Address: n.Config.P2P.ListenAddresses,
+		Network: n.Config.P2P.Network,
+	})
```

Comment:
I think having a constructor (e.g. `NewStatusHandler(listenAddrs []string, network string)`) even though the fields are public and all makes it clearer how to use the status handler.